contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -390,11 +390,55 @@ contract OETHVaultCore is VaultCore {
         }
     }
 
+    // @inheritdoc VaultCore
     function _allocate() internal override {
         // Add any unallocated WETH to the withdrawal queue first
         _addWithdrawalQueueLiquidity();
 
-        super._allocate();
+        uint256 wethAvailableInVault = _wethAvailable();
+        // Nothing in vault to allocate
+        if (wethAvailableInVault == 0) return;
+        uint256 strategiesValue = _totalValueInStrategies();
+        // We have a method that does the same as this, gas optimisation
```

Comment:
is this referring to `_totalValue`? That's different to `wethAvailableInVault + strategiesValue;`